### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/25e9fe42368f9d50d8f77676669ef154eeb3d72f

### CODE REVIEW

File: en/products/vpn/landing.ftl [hunk 1]
```diff
@@ -86,27 +86,47 @@ vpn-landing-faq-compatibility-question-desc-windows-v2 = <a href="{ $url }">{ -b
...
+# Outdated string
+# Variables:
+#   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/mac/
+vpn-landing-faq-compatibility-question-desc-mac-v3 = <a href="{ $url }">{ -brand-name-mac }</a> (10.14 and up)
+
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/mac/
 vpn-landing-faq-compatibility-question-desc-mac-v2 = <a href="{ $url }">{ -brand-name-mac }</a> (10.15 and up)
 
 # Outdated string
```
Comment (peiying2) [inline en/products/vpn/landing.ftl:98]: should we remove v1 string now?

File: en/products/vpn/landing.ftl [hunk 2]
```diff
@@ -86,27 +86,47 @@ vpn-landing-faq-compatibility-question-desc-windows-v2 = <a href="{ $url }">{ -b
...
+#   $url (url) - link to https://www.mozilla.org/products/vpn/mobile/android/
+vpn-landing-faq-compatibility-question-desc-android-v3 = <a href="{ $url }">{ -brand-name-android }</a> (version 8 and up)
+
+# Outdated string
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/mobile/android/
 vpn-landing-faq-compatibility-question-desc-android-v2 = <a href="{ $url }">{ -brand-name-android }</a> (version 6 and up)
 
 # Outdated string
 vpn-landing-faq-compatibility-question-desc-android = { -brand-name-android } (version 6 and up)
```
Comment (peiying2) [inline en/products/vpn/landing.ftl:111]: should we remove v1 string?

File: en/products/vpn/landing.ftl [hunk 3]
```diff
@@ -86,27 +86,47 @@ vpn-landing-faq-compatibility-question-desc-windows-v2 = <a href="{ $url }">{ -b
...
+# Variables:
+#   $url (url) - link to https://www.mozilla.org/products/vpn/mobile/ios/
+vpn-landing-faq-compatibility-question-desc-ios-v3 = <a href="{ $url }">{ -brand-name-ios }</a> (12.0 and up)
+
+# Outdated string
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/mobile/ios/
 vpn-landing-faq-compatibility-question-desc-ios-v2 = <a href="{ $url }">{ -brand-name-ios }</a> (13.0 and up)
 
 # Outdated string
```
Comment (peiying2) [inline en/products/vpn/landing.ftl:122]: remove v1 string?

**Review by peiying2 — CHANGES_REQUESTED** [review]
Should we remove v1 strings (including line #134)?

**Comment by alexgibson** [comment]
The v2 strings are still very new, so I think it makes sense to keep v1 for now.

I'll follow up with a fix for the wrongly placed L10n comment

**Review by peiying2 — APPROVED** [review]
Thanks for the update.

**Pull request merged**